instant/flags.vim

```diff
@@ -51,3 +51,11 @@ call s:plugin.Flag('clang_format_style', 'file')
 " "goimports" (https://godoc.org/golang.org/x/tools/cmd/goimports) to
 " additionally adjust imports when formatting.
 call s:plugin.Flag('gofmt_executable', 'gofmt')
+
+""
+" The path to the js-beautify executable.
+call s:plugin.Flag('js_beautify_executable', 'js-beautify')
+
+""
+" Default formatter for javascript files, 'clang_format' or 'js_beautify'.
+call s:plugin.Flag('javascript_formatter', 'clang_format')
```

Comment:
mb we can do smth like that?
```viml
call s:plugin.Flag(
  'javascript_formatter',
  executable(s:plugin.Flag('clang_format_executable')) ? 'clang_format' : 'js_beautify'
)
```


and about tests, i will wait until #21 will pass (to have a proper working example) and then will add the tests to this PR